src/longopt/grd2cpt_inc.h

```diff
@@ -25,70 +25,46 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'H', "",
-	          "",                  "",
+	{ 0, 'A', "alpha",
 	          "",                  "",
+	          "a",                 "all",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	GMT_C_CPT_KW,
+	{ 0, 'D', "bg|background",
+	          "i,o",               "in,out",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
+	{ 0, 'E', "nlevels",
 	          "",                  "",
+	          "c,f",               "cumulative,file",
 		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'F', "format|color_model",
+	          "c,g,h,r,R,x",       "cmyk,gray,hsv,rgb,name,hex",
+	          "c,k",               "categorical,key",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
+	{ 0, 'G', "truncate",          "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'H', "stdout",            "", "", "", "", GMT_TP_STANDARD },
```

Comment:
`stdout` is not a good name (programmers slang). Specially for wrappers where the concept does not make much sense.
Don't recall why I didn't implement this option in GMT.jl. Maybe `cptfile`?

Reply:
How about savecpt?

Reply:
`savecpt` looks good